src/App.js

```diff
@@ -44,22 +46,21 @@ class App extends Component {
     })
 
     if (data) {
-      console.log('data:', data)
       this.setState({user: data.user})
+      this.getRatings(this.state.user.id)
     }
   }
 
-  postUserRating = (userId, movieId, userRating) => {
-    this.postRating(userId, movieId, userRating)
-    .then(data => this.setState({userRatings: data.userRating}))
+  postUserRating = async (userId, movieId, userRating) => {
```

Comment:
I'm wondering if we should finalize on sticking with either async/await vs using .then, I'm not sure how that looks when we are using both in the same app or even if there's a real difference in between them